npc/other/divorce.txt

```diff
@@ -211,7 +211,8 @@ nif_in,190,112,5	script	Deviruchi#divorce	4_DEVIRUCHI,{
 		specialeffect2 EF_DEVIL;
 		wedding_sign = 0;
 		Zeny -= 2500000;
-		percentheal -100,-100;
+		percentheal -0,-100;
```

Comment:
was just a typo. And you are right, first i wanted to keep the percentheal to reduce Sp but it doesn't really make sense.